src/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// PHP_CodeSniffer does not define an autoloader in its composer.json,
+// so we must invoke it ourselves.
+require __DIR__ . "/../vendor/squizlabs/php_codesniffer/autoload.php";
```

Comment:
Yeah, I'm a bit worried we are coupling ourselves to the CodeSniffer autoloader here. So far, I only had problems with it and passing CodeSniffer a string to format while we already have an AST for the document is less than ideal